bundles/org.openhab.binding.kvv/src/main/java/org/openhab/binding/kvv/internal/DepartureResult.java

```diff
@@ -0,0 +1,315 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.kvv.internal;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * Represents the result of a call to the KVV api to fetch the next departures at a specific stop.
+ *
+ * @author Maximilian Hess - Initial contribution
+ *
+ */
+@NonNullByDefault
+public class DepartureResult {
+
+    /** timestamp */
```

Comment:
These comments are well-intentioned, but they are also redundant and lengthen the code. Since this is a DTO, no comments are needed at all.

Reply:
Solved.